x-pack/plugins/index_lifecycle_management/public/extend_index_management/components/add_lifecycle_confirm_modal.tsx

```diff
@@ -26,9 +28,25 @@ import {
 import { loadPolicies, addLifecyclePolicyToIndex } from '../../application/services/api';
 import { showApiError } from '../../application/services/api_errors';
 import { toasts } from '../../application/services/notification';
+import { PolicyFromES } from '../../application/services/policies/types';
+
+interface Props {
+  indexName: string;
+  closeModal: () => void;
+  index: any;
+  reloadIndices: () => void;
+  getUrlForApp: ApplicationStart['getUrlForApp'];
+}
+
+interface State {
+  selectedPolicyName: string;
+  selectedAlias: string;
+  policies: PolicyFromES[];
+  policyError?: string;
```

Comment:
nit: might be more clear as `policyErrorMessage`. I know this was pre-existing, so feel free to ignore 😄